subprojects/diagnostics/src/main/java/org/gradle/api/tasks/diagnostics/ProjectReportTask.java

```diff
@@ -74,17 +95,17 @@ protected void generate(Project project) {
     }
 
     private void render(final Project project, GraphRenderer renderer, boolean lastChild,
-                        final StyledTextOutput textOutput) {
+                        final StyledTextOutput textOutput, String projectSuffix) {
         renderer.visit(styledTextOutput -> {
-            styledTextOutput.text(StringUtils.capitalize(project.toString()));
+            styledTextOutput.text(StringUtils.capitalize(project.toString()) + projectSuffix);
             if (GUtil.isTrue(project.getDescription())) {
                 textOutput.withStyle(Description).format(" - %s", project.getDescription());
```

Comment:
Don't feel obligated, but since you're already messing with this file, do you want to fix #5746?

When a description contains newlines it prints wacky. It would be nice if the newlines indented nicely.